<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LP-5CD6518TRT" ignored="0" name="retry test" tests="8" failures="2" timestamp="22 May 2018 09:59:28 GMT" time="0.069" errors="0">
  <testcase name="a" time="0.009" classname="retryScenarios.test"/>
  <testcase name="b" time="0.002" classname="retryScenarios.test">
    <skipped/>
  </testcase> <!-- b -->
  <testcase name="b" time="0.002" classname="retryScenarios.test">
    <skipped/>
  </testcase> <!-- b -->
  <testcase name="b" time="0.002" classname="retryScenarios.test">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at retryScenarios.test.b(test.java:18)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- b -->
  <testcase name="c" time="0.002" classname="retryScenarios.test"/>
  <testcase name="d" time="0.001" classname="retryScenarios.test">
    <skipped/>
  </testcase> <!-- d -->
  <testcase name="d" time="0.002" classname="retryScenarios.test">
    <skipped/>
  </testcase> <!-- d -->
  <testcase name="d" time="0.002" classname="retryScenarios.test">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at retryScenarios.test.d(test.java:32)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- d -->
</testsuite> <!-- retry test -->
